modules/ROOT/pages/mule-oauth-provider-landing-page.adoc

```diff
@@ -119,25 +119,22 @@ You can define scopes in three different places:
 ...
 ----
 
-Full https://anypoint.mulesoft.com/exchange/org.mule.templates/api-gateway-oauth2-provider/[Mule OAuth 2.0 server for Anypoint Platform] asset available on Exchange.
+For more information, see https://anypoint.mulesoft.com/exchange/org.mule.templates/api-gateway-oauth2-provider/[Mule OAuth 2.0 server for Anypoint Platform].
 
 == Client Application Cache
 
-To prevent downtime of the Mule OAuth 2.0 provider due to errors when connecting to the platform,
-the Mule OAuth Client Store caches each valid Client Application for which a token is requested.
+To prevent any possible downtime of the Mule OAuth 2.0 provider, the Mule OAuth Client Store caches each valid Client Application for which a token is requested. The downtime can occur due to errors when connecting to Anypoint Platform.
 
-If the Mule OAuth 2.0 provider requests a new token while there is no connection, the OAuth 2.0 Client Store falls back to the cache.
+If the Mule OAuth 2.0 provider requests a new token during no connection, the OAuth 2.0 Client Store falls back to the cache. The entries expire in 30 days, which is the default setting. To ensure service availability in case of a restart or lack of connection to Anypoint Platform, you can configure this range to persist the cache locally.
 
-The expiration of its entries has a default of 30 days. Additionally, you can configure this range, and even set
-it to persist the cache locally to ensure service availability in case of a restart or lack of connection to the platform. In case
-a Client Application for which a token already exists in the Client Store is removed from Anypoint Platform, on the next token request,
-the Client Store will remove the Client from the cache, mirroring the platform status.
+In case a client application for which a token already exists in the client store is removed from Anypoint Platform,
+the client store removes the client from the cache on the next token request, mirroring the platform status. The mechanism that caches the result of a successful token validation performed by the xref:external-oauth-2.0-token-validation-policy.adoc[Mule OAuth 2.0 access token enforcement policy].
```

Comment:
Corrected comment unclear. Comment should be somthing like: The mechanism that caches the result of a successful token validation ***is part of*** the xref:external-oauth-2.0-token-validation-policy.adoc[Mule OAuth 2.0 access token enforcement policy].

Reply:
typo: Is went missing. fixed now.